shinydashboard/lantern/modules/smartresponsemodule.R

```diff
@@ -59,15 +59,15 @@ smartresponsemodule <- function(
     if (sel_fhir_version() != ui_special_values$ALL_FHIR_VERSIONS) {
       res <- res %>% filter(fhir_version == sel_fhir_version())
     }
-    if (sel_vendor() != ui_special_values$ALL_VENDORS) {
+    if (sel_vendor() != ui_special_values$ALL_DEVELOPERS) {
```

Comment:
the "Vendor" on line 49 (above, I can't click on the actual line) should be "Developer"